package.json

```diff
@@ -15,5 +16,10 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "jest-localstorage-mock": "^2.4.0"
```

Comment:
`jest-localstorage-mock` is a drop in fix for `localStorage` use in a non-window environment